StudCity.Frontend/src/pages/NotFound/NotFound.js

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import Button from "../../UI/Button";
+
+function NotFound() {
+	const dark = "#453e35";
+	const light = "#b4b45d";
+	const background = "#FFFFF0";
+	return(
+		<div className={`w-full bg-[${background}] h-screen flex  relative overflow-hidden`}>
+				<div className={`absolute left-[-5%] top-3/4 w-[400px] h-[400px] bg-[${dark}] rounded-[50%] `}></div>
```

Comment:
user '-left[5%]' instead of 'left-[-5%]' as official docs suggests 
PS: in other places as well